### ISSUE TITLE
How to Add ignore_idx Support to Cross-Entropy Loss Efficiently

### ISSUE THREAD

**Issue by Li-dongyang** [Dao-AILab/quack#8]
How to add the ignore_idx functionality to cross entropy in the existing framework (like[torch.nn.functional.cross_entropy](https://docs.pytorch.org/docs/stable/generated/torch.nn.functional.cross_entropy.html)) while minimizing performance loss in H100 as much as possible?

Thank you for your guidance!!

**Comment by tridao** [comment]
Shouldn't be too hard, probably just 1-2 lines around here
https://github.com/Dao-AILab/quack/blob/b3292e05910f0535756cf210994253218555af0b/quack/cross_entropy.py#L423

**Comment by Li-dongyang** [comment]
> Shouldn't be too hard, probably just 1-2 lines around here
> 
> [quack/quack/cross_entropy.py](https://github.com/Dao-AILab/quack/blob/b3292e05910f0535756cf210994253218555af0b/quack/cross_entropy.py#L423)
> 
> Line 423 in [b3292e0](/Dao-AILab/quack/commit/b3292e05910f0535756cf210994253218555af0b)
> 
>  label = cute.Int32(mTarget[row])

Got it, thanks a lot!
